lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -413,6 +406,15 @@ class MetroItinerary extends NarrativeItinerary {
             {!mini && (
               <ItineraryGrid className="itin-grid" role="group">
                 {/* TODO: a11y: add aria-label to parent element */}
+                <InvisibleHeader as={expanded && 'h2'}>
```

Comment:
When expanded, the `<h2>` is not needed IMO. Indeed, just above it is an `<h1>` with the main mode summary (e.g. "Walk + Transit"). Alternately, this element could replace the `<h1>` above, what do you think?

Reply:
Good point, I went ahead and removed the previous `h1` and replaced it with this header, since it was more descriptive